analytics/views_website.py

```diff
@@ -221,14 +230,17 @@ def bibliometrics_list_received(request):
 
     if data['selected_journal_code']:
         journal = request.stats.articlemeta.journal(code=data['selected_journal_code'])
-        titles.append(journal.title)
-        titles.append(journal.abbreviated_title)
+        if not journal.title in titles:
```

Comment:
Não é necessário validar se já existe o titulo na lista.

A linha 245 já resolve a remoção de itens duplicados.
forms = set([i.strip() for i in titles if i])